### DISCUSSION TITLE
Fingerprinting Client Devices

### DISCUSSION THREAD

**Discussion by wifiman06060** [nzymedefense/nzyme#753] [Ideas]
Fingerprinting AP to is revolutionary; but can client devices (cell phones, tablets, laptops, etc) be fingerprinted as well?   This would be handy when cell phones randomize their MAC addresses when "probe-requesting".  Identifying, for example four random MACS, are actually one device by fingerprinting, would be so beneficial for a WIDS

**Comment by lennartkoopmann** [comment]
Thanks! Created #755, currently planned for v2.0.

**Reply by justinfreitag** [reply]
Other apps, Fing, etc. already do this quite well.

**Comment by harry667788** [comment]
Very coold feature!! Can it still be expected in v2.0?
